tests/api_resources/abstract/test_test_helpers_api_resource.py

```diff
@@ -16,9 +15,12 @@ def __init__(self, resource):
 
             def do_stuff(self, idempotency_key=None, **params):
                 url = self.instance_url() + "/do_the_thing"
-                headers = util.populate_headers(idempotency_key)
                 self.resource.refresh_from(
-                    self.resource.request("post", url, params, headers)
+                    self.resource.request(
```

Comment:
should this method become private?

Reply:
A private `_request` already exists. I plan to remove this public method in a follow-up PR, since it can be replaced with `_request_and_refresh` where it's used.